github3/pulls.py

```diff
@@ -254,6 +256,9 @@ def is_merged(self):
 
         :returns: bool
         """
+        if self.merged:
```

Comment:
I was thinking that if a pull is merged, `self.merged` will always equal `True` in the future (even reverting the pull doesn't seem to change that), so there wouldn't be a point in checking the API. However, if `self.merged` is `False`, you'd probably want to be able to check the API to see if it has been merged yet?

The name of the method (`is_merged`) is a bit problematic though, as it sounds more like a getter for the `merged` attribute than something that would actually fire of an API request.